design/components/layout/SDSidebar.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <nav
+    class="absolute inset-y-0 right-0 w-full max-w-md p-6 text-white bg-gray-800"
+  >
+    <a href="#" @click.prevent="close" class="inline-block mb-12">
```

Comment:
This should have better been a `<button>`. For accessibility, you could add `aria-label`: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute